docs/data-sources/cbr_policies.md

```diff
@@ -0,0 +1,131 @@
+---
+subcategory: "Cloud Backup and Recovery (CBR)"
+---
+
+# huaweicloud_cbr_policies
+
+Use this data source to get available CBR policies within HuaweiCloud.
+
+## Example Usage
+
+```hcl
+variable policy_id {}
+
+data "huaweicloud_cbr_policies" "test" {
+  policy_id = var.policy_id
```

Comment:
use `name` or `type` will be more suitable.

Reply:
fixed.